src/TCPClient.hs

```diff
@@ -75,29 +73,31 @@ tCPClient defaultPort hostString blobHash blobData =
    in runTCPClient (clientSettings port (U8S.fromString host)) app
   where
     app :: AppData -> IO Bool
-    app appdata =
-      runConduitRes $
-      blobData .|
-      bejnarkliClient
-        (do appSink appdata
-            liftIO (shutdown (fromJust (appRawSocket appdata)) ShutdownSend)
-            appSource appdata)
-        blobHash
+    app appdata
+      = runConduitRes
+      $ blobData
+     .| bejnarkliClient
+          ( do
+              appSink appdata
+              liftIO (shutdown (fromJust (appRawSocket appdata)) ShutdownSend)
+              appSource appdata
+          )
+          blobHash
```

Comment:
I do love lining up those operators don't I. I do think it makes the flow here easier to follow.